CHANGELOG.md

```diff
@@ -11,6 +11,16 @@ This project uses [*towncrier*](https://towncrier.readthedocs.io/) and the chang
 
 <!-- towncrier release notes start -->
 
+## [Infrahub - v1.0.10](https://github.com/opsmill/infrahub/tree/infrahub-v1.0.10) - 2024-12-19
+
+### Added
```

Comment:
I think we are missing something about the schema migrations being fixed in this release

Reply:
Taking a look